src/index.html

```diff
@@ -11,7 +11,146 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset id="first" class="fieldset">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="surname">Surname:</label>
+          <input
+            id="surname" type="text"
+            autocomplete="off" name="surname"
+          >
+        </div>
+        <div class="form-field">
+          <label for="name">Name:</label>
+          <input
+            id="name" type="text"
+            autocomplete="off" name="name"
+            maxlength="15"
+          >
+        </div>
+        <div class="form-field">
+          <label for="age">How old are You?</label>
+          <input
+            id="age" type="number"
+            min="1" max="100"
+            value="12" name="age"
+          >
+        </div>
+        <div class="form-field">
+          <label for="date">Full date of birth:</label>
+          <input
+            id="date" type="date"
+            placeholder="mm/dd/yyyy" name="date"
```

Comment:
Let's write the more meaningful names, not cleat what date. `dateOfBirthDate` will be better